tox.ini

```
@@ -1,6 +1,6 @@
 [tox]
 minversion=2.9.0
-envlist = py36,lint,docs
+envlist = py36,lint,docs,docs-linkcheck
```

Since we don't fail the build when `docs-linkcheck` fails, we shouldn't execute it by default

```suggestion
envlist = py36,lint,docs
```